src/components/search/concept-modal/concept-modal.js

```diff
@@ -221,10 +322,21 @@ export const ConceptModalBody = ({ result }) => {
               relatedConcepts[cdeId] = null
             }
           }))
+          await Promise.all(cdeIds.map(async (cdeId, i) => {
+            try {
+              relatedStudies[cdeId] = await loadRelatedStudies(cdeId)
+            } catch (e) {
+              // Here, we explicitly want to halt execution and forward this error to the outer handler
+              // if a related concept request was aborted, because we now have stale data and don't want to
+              // update state with it.
+              if (e.name === "CanceledError" || e.name === "AbortError") throw e
```

Comment:
Are errors other than these two silently ignored?

Reply:
They aren't "ignored", as in, the line immediately following this (333) handles the error. A null value indicates to components downstream that an error occurred. As the comment explains, a request cancellation is explicitly bubbled up  to avoid overwriting state with stale data.